File: UI/PuantajApp/src/app/Models/Tally.ts
export interface Tally{
  tallyGuidId?:string;
  name?:string;
  jobDate?: Date;
  FirstDateTally?: Date;
  FinishDateTally?: Date;
  countHour?: number;
  studentGuidId?: string;
}
export interface TallyFace{
  tallyGuidId?:string;
  name?:string;
  jobDate?: string;
  FirstDateTally?: string;
  FinishDateTally?: string;
  countHour?: number;
  studentGuidId?: string;
}

File: Entities/Concrete/Tally.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Entities.Concrete
{
    public class Tally
    {
        [Key]
        public Guid TallyGuidId { get; set; }
        public DateTime JobDate { get; set; }
        public DateTime FirstDate { get; set; }
        public DateTime FinishDate { get; set; }
        public int CountHour { get; set; }
        public Guid StudentGuidId { get; set; }
        public Student Student { get; set; }

    }
}
